Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -106,7 +106,7 @@ WarpX::AddSpaceChargeFieldLabFrame ()
 #endif
 
     // Allocate fields for charge
-    // Also, zero out the phi data
+    // Also, zero out the phi data - is this necessary?
```

Comment:
I believe that zeroing out phi is necessary, since it sets the initial value at the start of the multigrid iteration. I don't think that the MLMG solver initializes phi.

Reply:
Thinking about this more, you are correct, the zeroing of phi is not necessary here. It would only need to be done once at the start of the simulation. I'll handle this in a separate PR.

Reply:
Thank you. That could help a lot in speeding up the solve as well since if the field doesn't change much between steps, the solver would start almost converged already.